src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public static Colors getRandomColor() {
```

Comment:
Avoid using static methods in your solution. Consider creating an instance of `ColorSupplier` and calling `getRandomColor` on that instance instead.